### ISSUE TITLE
Q440 keyword type / key is not uppercase

### ISSUE THREAD

**Issue by fgimian** [pgjones/flake8-sql#7]
Hey there, really cool extension to flake8 😄 

I believe it is reporting some false positives when table field names are `type` or `key`.

e.g.

```
./elite/libraries/launchpad_builder.py:195:0: Q440 keyword type is not uppercase
./elite/libraries/launchpad_builder.py:222:0: Q440 keyword type is not uppercase
./elite/libraries/launchpad_builder.py:247:0: Q440 keyword type is not uppercase
./elite/libraries/launchpad_builder.py:271:0: Q440 keyword type is not uppercase
./elite/libraries/launchpad_builder.py:296:0: Q440 keyword type is not uppercase
./elite/libraries/launchpad_builder.py:343:0: Q440 keyword type is not uppercase
./elite/libraries/launchpad_builder.py:352:0: Q440 keyword key is not uppercase
./elite/libraries/launchpad_builder.py:372:0: Q440 keyword type is not uppercase
./elite/libraries/launchpad_builder.py:396:0: Q440 keyword key is not uppercase
./elite/libraries/launchpad_builder.py:472:0: Q440 keyword key is not uppercase
./elite/libraries/launchpad_builder.py:472:0: Q440 keyword key is not uppercase
```

And here are some related lines:

Lines 192 - 196:

```python
            cursor.execute('''
                INSERT INTO items (rowid, uuid, flags, type, parent_id, ordering)
                     VALUES (?, ?, 2, ?, ?, ?)
            ''', (group_id, generate_uuid(), Types.PAGE, root_parent_id, page_ordering + 1)
            )
```

Lines 348 - 353:

```python
        cursor.execute('''
            UPDATE dbinfo
               SET value = 1
             WHERE key = 'ignore_items_update_triggers'
        ''')
        self.conn.commit()
```

Any help is greatly appreciated 😄 
Fotis

**Comment by pgjones** [comment]
This can be quite common as SQL has a lot of reserved keywords that make for good column names. The plugin allows for this to be avoided by adding a `setup.cfg` [configuration](https://github.com/pgjones/flake8-sql#configuration) file containing,
```
[flake8]
sql-excepted-names=key
```

**Comment by fgimian** [comment]
Thanks so much @pgjones, I can confirm this works as explained 😄
